domain/servicefactory/testing/service.go

```diff
@@ -37,11 +39,26 @@ func (s *TestingServiceFactory) ControllerNode() *controllernodeservice.Service
 	return nil
 }
 
+// Model returns the model service.
+func (s *TestingServiceFactory) Model() *modelservice.Service {
```

Comment:
I'd love to kill this `TestingServiceFactory` with fire.

Reply:
Going to punt this comment as this PR doesn't need anymore scope change.